### DISCUSSION TITLE
New docs: Consider making the content centralized on the screen

### DISCUSSION THREAD

**Discussion by andreynering** [rails/rails#51377] [Docs]
Hi there,

The [new docs](https://edgeguides.rubyonrails.org/) looks clean, and I liked it!

I'm not a designer, so my input is not a professional take, but the first thing that crossed my mind when I saw it is that having the meaningful content that far on the left hurts the readability a bit (for widescreen monitors), specially for long reads. Most sites have the content on the center of the screen, and I think that's a good UX design.

Two things that could be tried out:

1) Have a larger padding on the left so the content is more close to the center of the screen. The right navigation could be way less wide IMO.
2) Move the navigation to the left of the screen, so the start of the content is close to the center.

(If you take [Docusaurus](https://docusaurus.io/docs) as an example, they managed to have a right navigation while still having the content in the center).

Regarding 2, I did a quick hack on the browser inspect just to have an idea on how it would look:

### Before

<img width="1989" alt="Screenshot 2024-03-21 at 13 42 01" src="https://github.com/rails/rails/assets/7011819/c7733240-77d9-4af0-a707-e107c5275488">

### After

*The navigation could still be less wide, IMO

<img width="1988" alt="Screenshot 2024-03-21 at 13 41 33" src="https://github.com/rails/rails/assets/7011819/89798524-e272-443d-ab8d-79f0ecf30db1">

/cc @AmandaPerino

**Comment by AmandaPerino** [comment]
Hi @andreynering thanks for this suggestion. We actually are workshopping that idea as a possibility for the main menu (not each Guide menu, which are also known as 'Chapters', which will remain for now on the right). 
Still TBD if this is the route we will take when we tackle the Main menu.

For now, we wanted to start with the smallest possible element (the template of a Guide) and work incrementally from there.

We also wanted to focus purely on _design_ and not moving things around too much. The current layout **structure** has been used and navigated (and loved) for 15 years. Moving main elements around right off the bat would probably go down in the community as well as a lead balloon. 

You'll be happy to hear that we also reduced the width of the Chapter navigation in [the most recent update](https://github.com/rails/rails/pull/51372/commits/402ce05d4ebb3018492b52bd3ead1eb3056b9fca).

**Reply by andreynering** [reply]
> You'll be happy to hear that we also reduced the width of the Chapter navigation in [the most recent update](https://github.com/rails/rails/pull/51372/commits/402ce05d4ebb3018492b52bd3ead1eb3056b9fca).

@AmandaPerino Thanks, I'll try once it goes live.

Moving the navigation to the left was just an idea, and I agree that keeping things similar to the existing site is a good idea.

The real annoyance that I wanted to let you know is the fact that the content is on the left, and it should be centered.

I know the new site is a WIP and will be improved. Thanks for responding! 🙂

**Reply by AmandaPerino** [reply]
The content will be more centered with the next merge. Coming really soon so you can see.

**Comment by EmanuelFacundo** [comment]
I agree, on my screen it wasn't such an interesting UX with the content aligned to the left, but I don't see a problem with the main menu remaining on the right, I think reducing the container size on larger screens is enough, but I'm not a designer.

New docs on my screen:
![image](https://github.com/rails/rails/assets/38675971/6c70f305-1cd3-4cc8-ae7d-68bf2b881722)

**Reply by AmandaPerino** [reply]
That change (reduced container that will remain more centered) has been made in the most recent PR, merging soon. :)

**Comment by AmandaPerino** [comment]
Hi again @andreynering and @EmanuelFacundo - circling back to say that in the past few updates we have addressed the centering of the main content. The wider the screen, the more central the content. Thanks again for your feedback!
